sys/darwin/placeholder.txt

```diff
@@ -0,0 +1,30 @@
+# Copyright 2021 syzkaller project authors. All rights reserved.
+# Use of this source code is governed by Apache 2 LICENSE that can be found in the LICENSE file.
+
+include <fcntl.h>
+include <sys/mman.h>
+
+resource fd[int32]: 0xffffffffffffffff, AT_FDCWD
+
+open(file ptr[in, filename], flags flags[open_flags]) fd
+close(fd fd)
+mmap(addr vma, len len[addr], prot flags[mmap_prot], flags flags[mmap_flags], fd fd, offset fileoff)
+munmap(addr vma, len len[addr])
+mknod(file ptr[in, filename], mode flags[mknod_mode], dev int32)
+
+syz_execute_func(text ptr[in, text[target]])
+syz_emit_ethernet(len len[packet], packet ptr[in, eth_packet])
```

Comment:
You said darwin does not support tap that we need. So why this one?

Reply:
I explained in the commit message, but maybe I should also leave a comment in the file? W/o these definitions I get unused function compiler warnings for those three functions. In a follow up commit I'm even adding the full freebsd definitions here for those calls.

It's true that darwin doesn't bring a tap driver, but there are third party drivers like the (abandoned) tuntaposx driver or it's fork in Tunnelblick. I tested with the tunnelblick one and it got issues, but it's on my todo list to revisit that when building an FOSS XNU image for testing on gcp.

Reply:
Then I would suggest to `#if !GOOS_DARWIN` these functions in executor.

Reply:
Done.